bika/lims/content/analysiscategory.py

```diff
@@ -19,6 +19,7 @@
 from zope.interface import implements
 import sys
 import transaction
+from bika.lims import deprecated
```

Comment:
@xispa @Espurna please bear in mind that this import must be on 'wip' anyways. It is missing from the previously created(and merged) PR and currently on 'wip', instance doesn't run without this import added manually.